services/useDBService.ts

```diff
@@ -0,0 +1,130 @@
+import { V1_init_database, V2_smt } from "./migrations";
+import { databaseName, databaseVersion, AddedSiteParams, SiteFeedParams} from "../constants";
+import { useCRUD, FeedManager } from ".";
+import { UserFeed, AddedSite, SiteFeed } from "../models";
+
+export const useDBService = () => {
+
+    const { insert, getByColumnName, getAll, deleteByID } = useCRUD();
+    const { feedConverter } = FeedManager();
+
+    async function initDatabase() {
+
+        const request = indexedDB.open(databaseName, databaseVersion);
+
+        request.onupgradeneeded = event => {
+            // uncomment to check versions
+            // alert(`upgrade is called. old version = ${e.oldVersion}, new version ${e.newVersion}`);
+
+            const db = request.result;
+            
+            switch(event.oldVersion){
+                case 0:
+                    V1_init_database(db);
+                case 1:
+                    V2_smt(db);
+            }
```

Comment:
I think we should add here `V2_smt`
```suggestion
            switch(event.oldVersion){
                case 0:
                    V1_init_database(db);
                     V2_smt(db);
                     break;
                case 1:
                    V2_smt(db);
                    break;
            }
```